src/01_clock_resolution.jl

```diff
@@ -21,8 +21,11 @@ function estimate_clock_resolution(n_samples::Integer = 10_000)
     for _ in 1:n_samples
         t1 = Base.time_ns()
         t2 = Base.time_ns()
-        Δt = t2 - t1
-        min_Δt = min(min_Δt, Δt)
+        # On Linux AArch64 it seems that t1 and t2 could be the same sometimes
+        if t2 > t1
+            Δt = t2 - t1
+            min_Δt = min(min_Δt, Δt)
+        end
     end
 
     min_Δt
```

Comment:
In the unlikely event that you run into the `t2 == t1` problem at every sample, this fix would cause the function to return `typemax(UInt)`, which seems strange. 

Obviously, the chance that this will return `typemax(UInt)` is miniscule with the default `n_samples` of `10,000`, but maybe it would be safer to retry the current iteration until a valid sample is obtained? E.g.

```julia
function estimate_clock_resolution(n_samples::Integer = 10_000, attempt_limit::Integer = 20)
    min_Δt = typemax(UInt)
    for _ in 1:n_samples
        t1 = Base.time_ns()
        t2 = Base.time_ns()
        attempts = 0
        while t1 == t2
            t1 = Base.time_ns()
            t2 = Base.time_ns()
            attempts += 1
            if attempts > attempt_limit
                error("zero displacement between successive calls to time_ns()")
            end 
        end
        Δt = t2 - t1
        min_Δt = min(min_Δt, Δt)
    end
    return min_Δt
end
```

The whole `attempt_limit` thing might be overkill, but you get the idea.